### ISSUE TITLE
[feature request] An option to get machine-readable output from `zenn article:new`

### ISSUE THREAD

**Issue by kyoh86** [zenn-dev/zenn-editor#63]
## Summary

Now `zenn article:new` puts `📄\e[32m<slug>.md\e[39m created.`.
But I want an output which can be parsed with pipe.

e.g.

```console
$ zenn article:new --no-pretty
ecba19abfa3c41371a4a
```

or

```console
$ zenn article:new --no-pretty
articles/ecba19abfa3c41371a4a.md
```

## Argument

- Option name
  - `--silent` like a `curl`
  - `--porcelain` like a `git status`
  - `--machine-readable` like Vagrant commands
  - `--pretty` (default: true)

- Output
  - filename `articles/ecba19abfa3c41371a4a.md`
  - slug `ecba19abfa3c41371a4a`

**Comment by kyoh86** [comment]
[arg](https://www.npmjs.com/package/arg) doesn't support "--no-(boolean flag)" format.
So `--pretty` (default: true) should not be in the options.

**Comment by catnose99** [comment]
I see,
it makes easy to do something like this.
https://zenn.dev/kyoh86/articles/c02511740d70365a1420

I will merge your PR.

**Comment by kyoh86** [comment]
@catnose99 thanks, it's fixed by #66

**Comment by catnose99** [comment]
@kyoh86 
New CLI has been released.
Upgrade with `npm install zenn-cli@latest` to reflect the changes.
